src/ray/gcs/gcs_client/test/gcs_client_test.cc

```diff
@@ -886,6 +885,7 @@ TEST_P(GcsClientTest, TestGcsTableReload) {
 
   // Restart GCS.
   RestartGcsServer();
+  RAY_CHECK_OK(gcs_client_->Connect(*client_io_service_));
```

Comment:
Why do we need to reconnect here? This test only works with Redis, so it should just work. The token should be the same as before.